resources/install.rb

```diff
@@ -3,7 +3,13 @@
 action :install do
   include_recipe 'yum-epel' if platform_family?('rhel', 'amazon')
 
-  package %w(haveged gnupg2)
+  gpg2_package_name = if platform?('opensuseleap')
+                        'gpg2'
+                      else
+                        'gnupg2'
+                      end
+
+  package %W(haveged #{gpg2_package_name})
```

Comment:
Can you please move this to `libaries/helpers.rb` as a `gpg2_packages` method?

Reply:
Done in 3d06ba3c64ccb44c2295ffba929ce41db93b7910

Reply:
I've refactored it a bit in ab2a708606e18ec917b8d5131239537456f3eee1. I think this is what you meant in the first place :sweat_smile: